File: src/main/Java/boj_step4/boj5597.java
import java.io.*;

public class boj5597 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int capacity = 30;
        boolean[] register = new boolean[capacity];
        capacity -= 2;
        while (capacity-- > 0) {
            int number = Integer.parseInt(reader.readLine());
            register[number - 1] = true;
        }
        for (int i = 0; i < register.length; i++)
            if (!register[i]) {
                writer.append(String.valueOf(i + 1));
                writer.newLine();
            }
        writer.close();
    }
}

File: src/main/kotlin/boj5597.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    var students = IntArray(31) {0}
    var cnt = 28
    while (cnt-- > 0) {
        val studentNo = r.readLine()!!.toInt()
        students[studentNo]++
    }
    for(i in 1 until students.size) if(students[i] == 0) w.write(i.toString() + "\n")
    w.close()
}